src/gui/input_mac.cpp

```diff
@@ -42,19 +43,45 @@ QString GetModifierPrefix(Qt::KeyboardModifiers mod) noexcept
 	return modprefix;
 }
 
+// Fixup Modifiers, Apples OPTION key
+Qt::KeyboardModifiers de_DE(Qt::KeyboardModifiers mod, QChar c) noexcept
+{
+ 	std::vector<QChar> removeAltCharList  { '[', ']', '|', '{', '}', '~', '@', '\'' };
+	if(mod & Qt::AltModifier) {
+		if (std::find(removeAltCharList.begin(), removeAltCharList.end(), c) != removeAltCharList.end()) {
+			mod &= ~Qt::AltModifier;
+		}
+	}
+	return mod;
+}
 QKeyEvent CreatePlatformNormalizedKeyEvent(
 	QEvent::Type type,
 	int key,
 	Qt::KeyboardModifiers mod,
-	const QString& text) noexcept
+	const QString& text,
+	QLocale* locale) noexcept
 {
+	
+	
+	
 	if (!text.isEmpty())
 	{
 		const QChar c{ text.at(0) };
 		if (c.unicode() >= 0x80 && c.isPrint()) {
 			mod &= ~Qt::AltModifier;
 		}
+		
+		// German/Austrian/Swiss MacOS QWERTZ keyboard, remove AltModifier (OPTION KEY)
+		if(locale != nullptr) {
```

Comment:
`if (locale)`

No need for `== nullptr` or `!= nullptr` in C++.

Reply:
done